### ISSUE TITLE
Having more than one way of determing the site is out of data - out of data message doesn't show for P4:T

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes#1721]
We're finished all the subject data but because the subject counts are a little off the site is showing retired images to everyone and doesn't have a message up on the site. previously it went too early. For the future, we'll either need to tweak the subject completed count to get this value correct. Perhaps there should be a way that if all subjects are completed that when a classifier queue is remade and it sees that it changes a flag status for the site

**Comment by camallen** [comment]
Is this still happening? My counts are:
``` bash
workflow.retired_subjects_count
=> 20122
workflow.subjects_count
=> 29171
```

**Comment by mschwamb** [comment]
Thanks for looking into this. It was over the weekend, but on Sunday before I took the site down the subject csv said all the data was classified and now message was showing.  I added new data which is the numbers you're seeing.

**Comment by camallen** [comment]
Makes sense now, thanks. This is a know bug and something we will fix in #1639 & #1571
